packages/rich-text/src/to-tree.js

```diff
@@ -2,24 +2,34 @@
  * Internal dependencies
  */
 
+import { getActiveFormats } from './get-active-formats';
 import { getFormatType } from './get-format-type';
 import {
 	LINE_SEPARATOR,
 	OBJECT_REPLACEMENT_CHARACTER,
-	ZERO_WIDTH_NO_BREAK_SPACE,
 } from './special-characters';
 
-function fromFormat( { type, attributes, unregisteredAttributes, object } ) {
+function fromFormat( { type, attributes, unregisteredAttributes, object, boundaryClass } ) {
```

Comment:
Documenting even the internal functions can be helpful, as it wasn't obvious to me that `boundaryClass` is a boolean indicating that the specific `data-rich-text-format-boundary` class should be applied, rather than the classname itself (string) we want to apply.